README.md

```diff
@@ -188,8 +188,8 @@ using `cmake -D BUILD_SHARED_LIBS=OFF ..`.
 - `SENTRY_BUILD_RUNTIMESTATIC` (Default: OFF):
   Enables linking with the static MSVC runtime. Has no effect if the compiler is not MSVC.
 
-- `SENTRY_LINK_PTHREAD` (Default: ON):
-  Links to the `pthread` library on unix targets.
+- `SENTRY_LINK_THREADS` (Default: ON):
```

Comment:
I wonder if we should keep the old config name for backwards compatibility here.

Reply:
Yes, I think we can. In 99% occurrences thread library will be `pthread`.